.github/workflows/tests.yaml

```diff
@@ -255,6 +255,7 @@ jobs:
       S3METADATA: mongodb
       S3KMS: file
       S3_LOCATION_FILE: /usr/src/app/tests/locationConfig/locationConfigTests.json
+      S3_VERSION_ID_ENCODING_TYPE: base62
```

Comment:
should we keep this "unset" in mongo-v0, to exercise both behaviors?
(or actually just remove mongo-v0, as we always use v1 with mongo)

Reply:
The code base still supports both v0 and v1 so we should keep the tests. I made v0 tests run with hex versionIDs